### ISSUE TITLE
remap upgrade 

### ISSUE THREAD

**Issue by murphyg** [gsflow/gsflow-arcpy#38]
To eliminate many remap issues, remap files should be .csv files rather than ASCII, and the Arc function will then be reclass by table instead of reclass by ASCII.

**Comment by cgmorton** [comment]
@murphyg I thought you mentioned that you got this working and I was curious if you could create a branch with these changes?  This seemed like a pretty useful fix since the remap tables were so temperamental between the various ArcGIS version.

**Comment by murphyg** [comment]
Yes it looks like some good bug fixes are being suggested, I will address
these when I get a chance, probably some time next week.
Murphy
